tests/common/docker-compose-ssh-server.yaml

```diff
@@ -0,0 +1,23 @@
+version: "2.1"
+services:
+  scitacean-test-ssh-server:
+    image: lscr.io/linuxserver/openssh-server:latest
+    container_name: openssh-server
+    hostname: scitacean-test-ssh-server
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=CET  # Not UTC on purpose to test timezone detection
+#      - PUBLIC_KEY=yourpublickey #optional
+#      - PUBLIC_KEY_FILE=/path/to/file #optional
+#      - PUBLIC_KEY_DIR=/path/to/directory/containing/_only_/pubkeys #optional
+#      - PUBLIC_KEY_URL=https://github.com/username.keys #optional
+      - SUDO_ACCESS=false
+      - PASSWORD_ACCESS=true
+      - USER_NAME=the-scitacean
+      - USER_PASSWORD=sup3r-str0ng
+#    volumes:
+#      - /path/to/appdata/config:/config
```

Comment:
Will it be easier to use `.env` file instead of updating `docker-compose` file?
It seems like `volumes` can also use the environment variables defined in the `.env` files.

Like this: https://docs.docker.com/compose/environment-variables/set-environment-variables/

Reply:
Not sure. I would still have to generate the env file with paths. Given that the docker-compose file is yaml, I found it quite easy to modify it.
Or am I missing something?

Reply:
It's just general preference that I like separating parts that are consistent and the other parts that are optional or often modified : D.... If it'll only cause more works, then not necessary I think